middleware/cmd/middleware/main.go

```diff
@@ -53,7 +53,10 @@ func main() {
 		}
 	}
 	defer logBeforeExit()
-	middleware := middleware.NewMiddleware(argumentMap, false)
+	middleware, err := middleware.NewMiddleware(argumentMap, false)
+	if err != nil {
+		log.Fatal("error starting the middleware, is redis connected? \nIf you are running the middleware outside of the base consider setting the redis mock flag to true.")
```

Comment:
Then let's also add the flag to the top of the file to be able to start the middleware with mocked redis?

Reply:
Mind if I do this in a separate pr, since it is not directly related?

Reply:
Sure, no problem. I can just change it manually when running locally for now